src/formats/jpeg/errors.zig

```diff
@@ -0,0 +1,2 @@
+//! This module should define everything that is needed
+//! to describe the errors within the JPEG module.
```

Comment:
Remove it from now if you don't use it yet, you can add it back when it is required